composer.json

```diff
@@ -70,7 +70,7 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "yii\\log\\tests\\": "vendor/yiisoft/log/tests",
+            "Yii\\Log\\Tests\\": "vendor/yiisoft/log/tests",
```

Comment:
Why do we need to autoload `\Yii\Log\Tests` namespace in RBAC package?

Reply:
In fact, it is needed after all.

Travis errors with: 
```
yii\rbac\tests\unit\MySQLManagerCacheTest::testCheckAccessCache
Error: Class 'Yii\Log\Tests\ArrayTarget' not found
```

The test in this module depends on a test from the log module, and the `Yii\Log\Tests` namespace doesn't get registered by yiisoft/log since it's in its autoload-dev section

I'll reopen a PR.